RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -163,17 +163,94 @@ class RNUnifiedContacts: NSObject {
 
       callback([error.localizedDescription, NSNull()])
     }
-
   }
-
-
-
-
-
+  
+  @objc func createContact(contactData: NSDictionary, callback: (NSObject) -> () ) -> Void {
+    
+    let mutableContact = CNMutableContact()
+    let contactStore = CNContactStore()
+    let req = CNSaveRequest()
+    
+    // TODO:
+    // Extend method to handle more fields
+    //
+    //
+    mutableContact.givenName = contactData["givenName"] as! String
+    mutableContact.familyName = contactData["familyName"] as! String
+    mutableContact.organizationName = contactData["organizationName"] as! String
+    
+    for number in contactData["phoneNumbers"] as! NSArray {
+      let transformedNumber = transformNumber( number as! NSDictionary )
+      mutableContact.phoneNumbers.append( transformedNumber )
+    }
+    
+    for email in contactData["emailAddresses"] as! NSArray {
+      let transformedEmail = transformEmail ( email as! NSDictionary )
+      mutableContact.emailAddresses.append( transformedEmail )
+    }
+    
+    do {
+      req.addContact(mutableContact, toContainerWithIdentifier:nil)
+      try contactStore.executeSaveRequest(req)
+      print("Successfully created contact")
+      callback( [NSNull(), ""] )
```

Comment:
Interesting, I'm surprised that `addContact` or `executeSaveRequest` don't return the Contact or an identifier. I did some reading on the iOS docs and it looks like there's no way to get the identifier when you create a new Contact. That's bizarre.

So, I guess we can't return the identifier or the Contact object from here so instead let's do this: 

1. Success: `callback( [NSNull(), true] )`

2. Failure: `callback( [error.localizedDescription, false] )`

And then at least we can use it like `addContact( function(error, successful) {} )`